tests/test_flight.py

```diff
@@ -760,6 +761,18 @@ def test_ground_trajectory() -> None:
     )
     assert next(flight_iterate, None) is None
 
+def test_DME_NSE_computation() -> None:
+    flight = zurich_airport["EDW229"]
+    dme_zone = navaids.query("type == 'DME'").extent(flight)
+    dmes = dme_zone.query('not description.str.endswith("ILS")')
+    dmes
+    expected = pd.DataFrame(
+        [[0.29309839759917666, "TRA_ZUE"], [0.279227376079405, "KLO_TRA"]],
```

Comment:
It's probably better practice to compare floats rather than go for equality (with that level of precision which will go wrong at some point...)